agents/pfeiffer_tpg366/pfeiffer_tpg366_agent.py

```diff
@@ -53,7 +53,6 @@ def read_pressure(self, ch_no):
         self.comm.send(ENQ.encode())
         read_str = self.comm.recv(BUFF_SIZE).decode()
         pressure_str = read_str.split(',')[-1].split('\r')[0]
-        pressure = float(pressure_str)
         return pressure
```

Comment:
`pressure` no longer exists for you to return.

Reply:
Fixed, don't know how that deletion happened.